### ISSUE TITLE
Command `voyager:admin --create` fails during MySQL connection is strict

### ISSUE THREAD

**Issue by yoramdelangen** [thedevdojo/voyager#964]
- Laravel Version: 5.4
- Voyager Version: 0.11.7
- PHP Version: 7.0.14
- Database Driver & Version: MySQL Ver 15.1 Distrib 10.1.20-MariaDB, for osx10.12 (x86_64) using readline 5.1

### Description:
When creating an admin via command-line it only asks for a name and password. The database structure does not allow null values in the email field. In case the MySQL database connection is set strict it will cause the command to crash with an error: 
```
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'email' cannot be null
// and 
[PDOException]
  SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'email' cannot be null
```

Therefor are 2 ways to resolve this:
- Checking if the connection is strict and ask for the email(preferred I guess)?
- Not allowing strict connections.

I will create a PR with the first option to fix it properly.

### Steps To Reproduce:
Make sure the config item `database.connections.mysql.strict` is set to `true` before installation. 

Run the command `voyager:admin --create` on the command-line and proceed the name and password steps.

### EDIT
I found out there is an email argument that fills the email. I will add an extra check in case there isn't one set and adding documentation for it.

**Comment by nunomsh** [comment]
The email is create in the comand, so you don't need to anything. Pass on the email it work fine

 php artisan voyager:admin demo@demo.com --create

**Comment by abdgad** [comment]
Fixed in #965
